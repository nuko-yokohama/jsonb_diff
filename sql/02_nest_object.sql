SELECT * FROM jsonb_diff(
  '{"id":1, "obj":{"key":1, "data1":"foo", "data2":"bar"}}',
  '{"id":1, "obj":{"key":1, "data1":"foo", "data2":"baz"}}'
);
