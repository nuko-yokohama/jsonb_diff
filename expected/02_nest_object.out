SELECT * FROM jsonb_diff(
  '{"id":1, "obj":{"key":1, "data1":"foo", "data2":"bar"}}',
  '{"id":1, "obj":{"key":1, "data1":"foo", "data2":"baz"}}'
);
NOTICE:  table "scm_table" does not exist, skipping
  kind  |    left_path     |    left_schema    |    right_path    |   right_schema    
--------+------------------+-------------------+------------------+-------------------
 update | ->'obj'->'data2' | {"string": "bar"} | ->'obj'->'data2' | {"string": "baz"}
(1 row)

